src/index.html

```diff
@@ -138,7 +138,7 @@ <h1 class="description__title">Большой театр</h1>
           <button class="description__button fs18">Афиша</button>
         </div>
         <div class="theater__carousel carousel fs18">
-          <div class="carousel__preview">
+          <div class="carousel__preview js-preview">
```

Comment:
```suggestion
          <div class="carousel__preview js-carousel-preview">
```
I think it will be better